config/config.go

```diff
@@ -187,6 +187,7 @@ func newCGRConfig(config []byte) (cfg *CGRConfig, err error) {
 			ProfileIDs:           []*utils.DynamicStringSliceOpt{},
 			ProfileIgnoreFilters: []*utils.DynamicBoolOpt{},
 			RoundingDecimals:     []*utils.DynamicIntOpt{},
+			PrometheusMetrics:    []*utils.DynamicStringSliceOpt{},
```

Comment:
Need some better naming here since in the end are stat profiles and not really metrics we define.